### DISCUSSION TITLE
Remove upper constraint of 12.99 for Newtonsoft.Json dependency for UmbracoCms.Core

### DISCUSSION THREAD

**Discussion by DanDiplo** [umbraco/Umbraco-CMS#12231] [Features and ideas]
Currently [UmbracoCms.Core](https://www.nuget.org/packages/UmbracoCms.Core/) NuGet package (in Umbraco 8) has a dependency on `Newtonsoft.Json` expressed as:

[Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) (>= 12.0.1 && < 12.999999.0)

This prevents upgrading to the latest version of `Newtonsoft.Json` (currently `13.0.1`) and, more crucially, prevents installing any other dependencies that require a later version than Umbraco allows. 

For instance, I was trying to install the latest version of some Google APIs, but these all required 13.0.1 and Umbraco's dependency on this prevented installation. See https://www.nuget.org/packages/Google.Apis.Core/ for an example. This means you can never use any of Google's recent APIs with Umbraco 8. It's also possible that there may be improvements or security fixes in `Newtonsoft.Json` that Umbraco will prevent being installed.

Is there a good reason why this constraint can't be raised to 13.x ? If not, could it be removed for reasons outlined above. Thank you!

**Comment by nul800sebastiaan** [comment]
Sure, doesn't look like they're listing and real breaking changes so we can relax it: https://github.com/umbraco/Umbraco-CMS/issues/12235

We have no release date for 8.18.4 yet, for now you can install the google apis as usual when there's too many restrictions by ignoring them, use this in the Package Manager Console: `Install-Package Google.Apis.Core -IgnoreDependencies`

**Comment by Jogai** [comment]
Especially after this: https://github.com/advisories/GHSA-5crp-9r3c-p9vr 

Important for v7 too..
